hs/Response/DrawJson.hs

```diff
@@ -0,0 +1,66 @@
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
@R0ss don't make a new route for your work on this. You can modify the existing "Draw" page.